spec/factories/application_qualification.rb

```diff
@@ -6,18 +6,26 @@
     subject { Faker::Educator.subject }
     grade { %w[A B].sample }
     predicted_grade { %w[true false].sample }
-    start_year { Time.zone.today.year }
-    award_year { Faker::Date.between(from: 60.years.ago, to: 3.years.from_now).year }
+    award_year { qualifications_start_year + 3 }
     institution_name { Faker::University.name }
     institution_country { Faker::Address.country_code }
     equivalency_details { Faker::Lorem.paragraph_by_chars(number: 200) }
 
+    transient do
+      qualifications_duration { 6 }
```

Comment:
do we need all this if degree is the only one that accepts a start year? Could we instead define the dates there and avoid having a transient block?

Reply:
ended up with the transient block because all the candidates generate by the job ended up with the same year values, but perhaps not needed if the approach is changed 🤔 ?

Reply:
Yea I think to avoid complexity initially unless the need arises, we could try something like this: 
```
award_year { Faker::Date.between(from: 10.years.ago, to: 1.years.ago).year }
```
for generic applications, and for the ones that require a start year, we can do something similar as such:
```
start_year { Faker::Date.between(from: 5.years.ago, to: 3.years.ago).year  }
award_year { Faker::Date.between(from: 2.years.ago, to: 1.years.ago).year }
```